### DISCUSSION TITLE
Can vr players see other very players hand movements

### DISCUSSION THREAD

**Discussion by TecEash1** [QuestCraftPlusPlus/QuestCraft#376] [Q&A]
Hi, I was just wondering if mxcr core is installed on a fabric server, and 2 or more people that are running questcraft joined, would they be able to see each others hand movements/ any vr movements?

**Comment by TecEash1** [comment]
No they cannot, all it adds is dynamic player height, which they can't see and other stuff

**Reply by TecEash1** [reply]
I am from the future in 2023 - Here QuestCraft uses Vivecraft which supports this. You must be patient youngling

**Comment by Novampr** [comment]
Techy, are you sane?

**Comment by KendrickLamar2003** [comment]
> Techy, are you sane?

DEFINITELY INSANE

**Comment by TecEash1** [comment]
what did I do here...

Welp lets get this feature added guys!

**Comment by Novampr** [comment]
Yes
